packages/manager/src/views/WebPropertyDetailPage/components/SSR/ViewLogs.tsx

```diff
@@ -0,0 +1,137 @@
+import { fetchLogsforSpa, useListOfPods } from '@app/services/appLogs';
+import {
+  CodeBlock,
+  EmptyState,
+  EmptyStateIcon,
+  Select,
+  SelectOption,
+  Title
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  spaName: string;
+  propertyIdentifier: string;
+  env: string;
+  type: string | number;
+  idList: string[];
+};
+
+export const ViewLogs = ({ propertyIdentifier, spaName, env, type, idList }: Props) => {
+  const [logsData, setLogsData] = useState<any>([]);
+  const [isLogsLoading, setIsLogsLoading] = useState<boolean>(true);
+  const [isOpen, setIsOpen] = useState(false);
+  const [Id, setId] = useState('');
+
+  const podList = useListOfPods(propertyIdentifier, spaName, env);
+
+  const onToggle = (isOpen1: boolean) => {
```

Comment:
`isOpen1: boolean` make it to `isOpen: boolean` or any preferred name, isOpen1 is not looking good